code/drasil-code/lib/Language/Drasil/Code/Imperative/GOOL/ClassInterface.hs

```diff
@@ -52,6 +52,8 @@ type CaseName = String
 type ExamplePurpose = String
 -- | Description of example
 type ExampleDescr = String
+-- | Motivation of example
+type ExampleMotivation = String
```

Comment:
this information that is quite specific to the `README` has really gotten too large to fit in `Language.Drasil.Code.Imperative.GOOL.ClassInterface`. This should be split out into its own file.

It doesn't even belong to `Language.Drasil.Code.Imperative`, as it is quite independent of that. Finding a better home needs a design.